ui/raidboss/data/05-shb/alliance/the_puppets_bunker.txt

```diff
@@ -393,7 +393,7 @@ hideall "--sync--"
 6399.8 "Four Parts Resolve?" sync /:Compound 2P:519E:/
 
 # This is adjusted 9 seconds ahead so that Compound Pod: R012 lines up.  @_@;;
-6400.7 "--sync--" sync /:519B:Compound 2B/ window 50,50
```

Comment:
Wow, thanks for this! I don't know how this worked because I tested this timeline against this case several times.  (Maybe I introduced a typo later???) I re-tested with your change and it still looks good, so seems good to go.